dis_snek/models/discord/guild.py

```diff
@@ -216,6 +216,12 @@ def _process_dict(cls, data: Dict[str, Any], client: "Snake") -> Dict[str, Any]:
 
         if welcome_screen := data.get("welcome_screen"):
             data["welcome_screen"] = GuildWelcome.from_dict(welcome_screen, client)
+
```

Comment:
This should be remade into a property that returns dict[user_id, VoiceState]. Currently01, `Guild.voice_states` just returns the raw dict

![image](https://user-images.githubusercontent.com/38250010/160820907-ec887943-ef56-482f-94a4-d018a6b4dcd6.png)

Reply:
Thats not the fault of this code snippet. Caching a voice state is async, so it made more sense to do it in tasks to prevent bogging down when caching *busy* guilds. 

I agree a property should be made, adding it now

Reply:
Resolved in a nice ick way. 

8baddbf3a39c4bd1a0dbd4da3bd4eae5d79ebfb3